src/main/java/uk/gov/hmcts/ccd/domain/model/definition/SearchResultField.java

```diff
@@ -108,4 +106,16 @@ public String getDisplayContextParameter() {
     public void setDisplayContextParameter(String displayContextParameter) {
         this.displayContextParameter = displayContextParameter;
     }
+
+    public String getUseCase() {
+        return useCase;
+    }
+
+    public void setUseCase(String useCase) {
+        this.useCase = useCase;
+    }
+
+    public JsonNode getCaseFieldNode(JsonNode topLevelCaseFieldNode) {
+        return CaseFieldPathUtils.getCaseFieldNodeByPath(topLevelCaseFieldNode, getCaseFieldPath());
```

Comment:
can you inline this method? i.e. remove it and call directly the CaseFieldPathUtils in the current invoker of this method